app/assets/stylesheets/locals/plain.css.scss

```diff
@@ -10,5 +10,11 @@
             display: none;
             // Title at top of transcript isn't necessary: We know what we're looking at.
         }
+        .play-from-here::before {
+            content: '\25B6'; // right-pointing triangle
+            cursor: pointer;
+            color: $dark-blue;
+            padding-right: 0.3em;
+        }
```

Comment:
@afred: I didn't want to put the arrow glyph in the actual html: If I copy the text from my screen, I don't want those arrows.

As best I understand, the CSS content only works on ::before or ::after.